dotnet/src/SemanticKernel/Orchestration/Plan.cs

```diff
@@ -16,29 +19,72 @@ namespace Microsoft.SemanticKernel.Orchestration;
 /// Standard Semantic Kernel callable plan.
 /// Plan is used to create trees of <see cref="ISKFunction"/>s.
 /// </summary>
-public class Plan : ISKFunction
+public sealed class Plan : ISKFunction
 {
+    /// <summary>
+    /// Converter for <see cref="ContextVariables"/> to/from JSON.
+    /// </summary>
+    public class ContextVariablesConverter : JsonConverter<ContextVariables>
```

Comment:
I would remove this to a different file, having a general converter seems to be desirable out of plan contexts as well.

Reply:
Agreed, moving out.

Reply:
Appreciate @dluc your feedback on where to move things around re: projects + namespaces